contrib/storage-hive/hive-exec-shade/pom.xml

```diff
@@ -158,6 +158,8 @@
                  you can use TestHiveStorage.readFromAlteredPartitionedTableWithEmptyGroupType() test case. -->
                 <exclude>org/apache/parquet/**</exclude>
                 <exclude>shaded/parquet/org/**</exclude>
+                <exclude>org/apache/commons/lang/**</exclude>
```

Comment:
I'm afraid it can break something since hive explicitly includes these libraries into `hive-exec` jar: https://github.com/apache/hive/blob/master/ql/pom.xml#L958.

As an alternative solution, I would recommend relocating them (as it is done above for other libraries).